### DISCUSSION TITLE
Your module has been copied.

### DISCUSSION THREAD

**Discussion by ShaTang-debug** [ykundesu/SuperNewRoles#1271] [General]
Module author, are you there?
I am a player from China.There is a game in our country that has stolen your module content without permission.Not only your module is stolen, but also other people's modules.The name of this game is "Super sus"
He is still stealing your module content.

**Comment by ykundesu** [comment]
Yes, it's me. Could you please provide a link?

**Reply by ShaTang-debug** [reply]
https://space.mztgame.cn

**Reply by ShaTang-debug** [reply]
The English link is Supersus/io.en

**Reply by ykundesu** [reply]
If this mod's code or assets are not used, there is nothing I can do about it legally. I'm in trouble.

**Reply by ShaTang-debug** [reply]
The identities in this game are stolen from your modules.

**Reply by ykundesu** [reply]
Id？Do you have any screenshots?

**Reply by ShaTang-debug** [reply]
Our laws in China are different. You have the right to sue this game for infringing your copyright.

**Reply by ykundesu** [reply]
I can't tell without screenshots.

**Reply by ShaTang-debug** [reply]
I don't know how to send screenshots, but there are other links.

**Reply by ykundesu** [reply]
What's the link?

**Reply by ykundesu** [reply]
Even under Chinese law, an idea should not be protected by copyright.
Copyright is concrete. For example, it applies to source code, images, sounds, etc. Do it have those?

**Reply by ShaTang-debug** [reply]
He has source code and images.

**Reply by ykundesu** [reply]
So where is it. I couldn't find it. Where is the specific link.

**Reply by ykundesu** [reply]
I forgot to mention that I am using a translation, so I apologize if the English is not correct.

**Reply by ShaTang-debug** [reply]
Supersus/io.en

**Reply by ShaTang-debug** [reply]
This is the link.

**Reply by ShaTang-debug** [reply]
Sorry, the link should be Supersus.io/en

**Reply by ykundesu** [reply]
About the name? I do not know what it is. Which is it?

**Reply by ShaTang-debug** [reply]
the name is Super sus
This is his international name.

**Reply by ykundesu** [reply]
Yes, I do.
Where's the specific part that's being plagiarized? I can't find it.

**Reply by ShaTang-debug** [reply]
The specific part is in his identity system.

**Reply by ykundesu** [reply]
What is that? Where is that code?

**Reply by ShaTang-debug** [reply]
I don't know where the code is, But I know which identities he copied.

**Reply by ykundesu** [reply]
Which one?

**Reply by ShaTang-debug** [reply]
He plagiarized
1.office worker
2.whistle blower
3.hunter
There are still some I am counting.

**Reply by ShaTang-debug** [reply]
I'm a little curious about how you want to solve it. It is precisely because this game has copied so many modules that it can be popular internationally.

**Reply by ykundesu** [reply]
Are those identities the sheriffs, etc.?

**Reply by ShaTang-debug** [reply]
Yes

**Reply by ykundesu** [reply]
Huh? Are you kidding me? It's just an idea. No source code or images are used. In other words, it is not copyrighted. So we have no control over it. Understand?

**Reply by ShaTang-debug** [reply]
But he used the source code.

**Reply by ykundesu** [reply]
Do you know what source code is?
Where's your proof?
Source code is the code that makes the program work.

**Reply by ShaTang-debug** [reply]
Our understanding is different. According to our laws, this is plagiarism.
